src/main/java/com/direwolf20/buildinggadgets/common/tools/ToolRenders.java

```diff
@@ -625,51 +470,123 @@ private static void renderBox(Tessellator tessellator, BufferBuilder bufferBuild
-        bufferBuilder.pos(maxX, maxY, minZ).color(red, green, blue, alpha).endVertex();
-        bufferBuilder.pos(maxX, minY, minZ).color(red, green, blue, alpha).endVertex();
+        bufferBuilder.pos(startX, startY, startZ).color(red, green, blue, alpha).endVertex();
+        bufferBuilder.pos(startX, endY, startZ).color(red, green, blue, alpha).endVertex();
+        bufferBuilder.pos(endX, endY, startZ).color(red, green, blue, alpha).endVertex();
+        bufferBuilder.pos(endX, startY, startZ).color(red, green, blue, alpha).endVertex();
 
         //west
-        bufferBuilder.pos(minX, minY, maxZ).color(red, green, blue, alpha).endVertex();
-        bufferBuilder.pos(maxX, minY, maxZ).color(red, green, blue, alpha).endVertex();
-        bufferBuilder.pos(maxX, maxY, maxZ).color(red, green, blue, alpha).endVertex();
-        bufferBuilder.pos(minX, maxY, maxZ).color(red, green, blue, alpha).endVertex();
+        bufferBuilder.pos(startX, startY, endZ).color(red, green, blue, alpha).endVertex();
+        bufferBuilder.pos(endX, startY, endZ).color(red, green, blue, alpha).endVertex();
+        bufferBuilder.pos(endX, endY, endZ).color(red, green, blue, alpha).endVertex();
+        bufferBuilder.pos(startX, endY, endZ).color(red, green, blue, alpha).endVertex();
 
         //south
-        bufferBuilder.pos(maxX, minY, minZ).color(red, green, blue, alpha).endVertex();
-        bufferBuilder.pos(maxX, maxY, minZ).color(red, green, blue, alpha).endVertex();
-        bufferBuilder.pos(maxX, maxY, maxZ).color(red, green, blue, alpha).endVertex();
-        bufferBuilder.pos(maxX, minY, maxZ).color(red, green, blue, alpha).endVertex();
-        
+        bufferBuilder.pos(endX, startY, startZ).color(red, green, blue, alpha).endVertex();
+        bufferBuilder.pos(endX, endY, startZ).color(red, green, blue, alpha).endVertex();
+        bufferBuilder.pos(endX, endY, endZ).color(red, green, blue, alpha).endVertex();
+        bufferBuilder.pos(endX, startY, endZ).color(red, green, blue, alpha).endVertex();
+
         //north
-        bufferBuilder.pos(minX, minY, minZ).color(red, green, blue, alpha).endVertex();
-        bufferBuilder.pos(minX, minY, maxZ).color(red, green, blue, alpha).endVertex();
-        bufferBuilder.pos(minX, maxY, maxZ).color(red, green, blue, alpha).endVertex();
-        bufferBuilder.pos(minX, maxY, minZ).color(red, green, blue, alpha).endVertex();
+        bufferBuilder.pos(startX, startY, startZ).color(red, green, blue, alpha).endVertex();
+        bufferBuilder.pos(startX, startY, endZ).color(red, green, blue, alpha).endVertex();
+        bufferBuilder.pos(startX, endY, endZ).color(red, green, blue, alpha).endVertex();
+        bufferBuilder.pos(startX, endY, startZ).color(red, green, blue, alpha).endVertex();
         tessellator.draw();
     }
 
+    private static class Utils
+    {
+        static IBlockState getStartBlock(RayTraceResult lookingAt, EntityPlayer player)
+        {
```

Comment:
We usually put them in the same line, not?
I think it is even in the contribution guide.

Reply:
You're likely right. I should read it.